micro/example_test.go

```diff
@@ -28,33 +29,33 @@ func ExampleAddService() {
 	}
 	defer nc.Close()
 
-	echoHandler := func(req *Request) {
+	echoHandler := func(req micro.Request) {
 		req.Respond(req.Data())
 	}
 
-	config := Config{
+	config := micro.Config{
 		Name:        "EchoService",
 		Version:     "v1.0.0",
```

Comment:
this should be `1.0.0` or would get a validation error, maybe improve the regex to allow for either? Some packages that parse semver allow for either style too

Reply:
We agreed to use the official semver regex that does not allow for "v" in front, it not being part of semantic version.

Reply:
Yes, exactly. In this case, the example is broken, I'll fix it.